tests/converters/velobrix_test.py

```diff
@@ -0,0 +1,64 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+from pathlib import Path
+from unittest.mock import Mock
+
+import pytest
+from requests_mock import Mocker
+
+from parkapi_sources.converters.velobrix import VelobrixPullConverter
```

Comment:
The imports at src/parkapi_sources/parkapi_sources.py and /src/parkapi_sources/converters/__init__.py are missing.